dependency_support/com_github_westes_flex/bundled.BUILD.bazel

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 Google LLC
```

Comment:
It's annoying that GitHub doesn't seem to syntax highlight this file :-(

Reply:
This is a very blind guess, but maybe using `.gitattributes` as in https://github.com/im-tomu/fomu-workshop/pull/390 does the trick?

Reply:
I tried adding `.gitattributes` but it doesn't seem to have an effect. Maybe it needs to be merged first?

Reply:
Effective or not, it is harmless and it's always good to be explicit about these "metadetails". Hence, let's leave it.

Anyway, linguist is meant for the repo language usage statistics. The effect on syntax hightlighting was just something I hoped that might work.